### PR TITLE
Tour improvements

### PR DESCRIPTION
**Pull request by gvdongen**
- Make the exposed ports in docker run uniform 
- Improve explanation of service registration
- Improve explanation of keyed services

### CODE REVIEW

File: docs/tour.mdx [hunk 1]
```diff
@@ -85,21 +85,36 @@ When you are done with the tutorial, stop the runtime with `docker stop restate_
 
 To do resilience experiments later on, you can restart the runtime (without losing state) with `docker restart restate_dev`.
 
-Discover the services:
+Register the services:
+
+Now, we need to tell Restate where the service runs.
```
Comment (AhmedSoliman) [inline docs/tour.mdx:90]:
```suggestion
Now, we need to tell the Restate runtime where the service runs.
```
for consistency, if you chose to pick this suggestion from #159

File: docs/tour.mdx [hunk 2]
```diff
@@ -174,7 +193,8 @@ The `app.ts` file contains the definition of the Restate server that serves the
 Two types of Restate services exist:
 1. **Keyed service**:
 All service invocations are sharded on a user-defined key.
-Per key there is at most one concurrent invocation.
+The scope of a key is on a per-service basis.
+Per key there is at most one concurrent invocation at the service level, so across all functions in that service.
```
Comment (AhmedSoliman) [inline docs/tour.mdx:197]: Maybe more Layman's terms? like
"Function invocations of keyed services are serialized on the user-defined key. Therefore, at most one function can run at a time on a given key within such service."

I don't know if this is better, but what do you think?

File: docs/tour.mdx [hunk 3]
```diff
@@ -219,6 +239,12 @@ The functions of a keyed router have three optional parameters
 
 You can choose the parameter names differently.
 
+As mentioned, for keyed services there can be at most one concurrent invocation at the service level.
+So this means that if there is a call to `hello()`, then a call to `callMe()` will be queued
```
Comment (AhmedSoliman) [inline docs/tour.mdx:243]:
```suggestion
So this means that if there is a call to `hello()`, then a call to `callMe()`, then `callMe()` will wait in line until `hello()` has finished, if and only if both were invoked with the same value of `key`.
```

**Review by AhmedSoliman — APPROVED** [review]
A few minor comment, thanks for working on this @gvdongen

File: docs/tour.mdx (hunk 1 above)
Comment (gvdongen) [reply docs/tour.mdx:90]: We once agreed on calling the runtime just "Restate" in the docs.

File: docs/tour.mdx (hunk 2 above)
Comment (gvdongen) [reply docs/tour.mdx:197]: Sounds good!

**Pull request merged**